core/jazz_metrics/config/global-config.json

```diff
@@ -7,5 +7,16 @@
     "{conf-apikey-dev}": "{conf_stack_prefix}-dev",
     "{conf-apikey-stg}": "{conf_stack_prefix}-stg",
     "{conf-apikey-prod}": "{conf_stack_prefix}-prod"
+  },
+  "STATISTICS_MAP": {
```

Comment:
Should we keep this under APIGEE object in the JSON because these are related to APIGEE?

Reply:
r u suggesting adding this to installer-vars?

Reply:
Nope, inside the config file under APIGEE key. 
```json
{
  "APIGEE": {
    "STATISTICS_MAP": {
    },
    "INTERVAL_MAP": {
    }
  }
}
```

Reply:
updated